src/components/Navigation.js

```diff
@@ -9,13 +9,13 @@ const Navigation = () => {
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <Link to="/">Home</Link>
+          <Link to={"/"}>Home</Link>
```

Comment:
```suggestion
<Link to="/">Home</Link>
```
(even though you know both)